style.css

```diff
@@ -0,0 +1,270 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  box-shadow: none;
+}
+
+body {
+  margin: 1rem;
+  padding: 1rem;
+  width: 327px;
+}
+
+.header_container {
+  padding: 0.5rem;
+  font-family: sans-serif;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 0.5rem;
+}
+
+.navbar img {
+  align-self: center;
+  height: 28px;
+  left: 12px;
+  top: 10px;
+}
+
+main {
+  background-image: url("/assets/header-illsutration-mobile@2x.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 100% 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.intro_section h1 {
+  color: #172b4d;
+  width: 327px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  height: 192px;
+  font-size: 2.5rem;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  align-self: center;
+  padding-top: 4rem;
+  margin-bottom: 2rem;
+}
+
+.intro_section span {
+  color: rgb(51, 169, 51);
+}
+
+.intro_section p {
+  width: 327px;
+  height: 200px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 25px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  align-self: center;
+  margin-top: 2rem;
+  margin-bottom: 1rem;
+  padding-top: 2rem;
+}
+
+main > div {
+  align-self: center;
+  margin: 1rem 0;
+
+  /* width: 40px;
+  height: 40px; */
+}
+
+main ul {
+  margin-top: 2.5rem;
+  list-style-type: none;
+  display: flex;
+  justify-content: center;
+}
+
+main ul a {
+  color: #505f79;
+  font-size: larger;
+}
+
+main ul li {
+  padding: 1rem;
+}
+
+section {
+  font-size: 30px;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+}
+
+section h2 {
+  font-size: 2.5rem;
+  margin-top: 2rem;
+}
+
+.underline {
+  width: 50px;
+  border: 2px solid #36b37e;
+  background: #36b37e;
+  border-radius: 10%;
+  margin-top: 0.4rem;
+  margin-bottom: 4rem;
+  align-self: center;
+}
+
+.section_container {
+  display: grid;
+  grid-template-columns: 327px;
+}
+
+.card {
+  height: 474px;
+  width: 327px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  margin: 0.5rem 0;
+  margin-left: -22px;
+}
```

Comment:
- [ ] Please make sure your project cards are centered as seen in the Figma design 🙏 
- [ ] Kindly give your project cards a border as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/185628543-07405df9-f0b0-48b9-a0b1-8589028cbd7d.png)  |  ![image](https://user-images.githubusercontent.com/74814780/185628806-a10c5f0b-d724-45fd-83d3-cfdfab07d817.png)